cmd/snap-fde-keymgr/main.go

```diff
@@ -34,23 +35,23 @@ import (
 
 var osStdin io.Reader = os.Stdin
 
-type commonDeviceMixin struct {
-	// TODO: support for multiple devices in the command line
-	Device string `long:"device" description:"encrypted device" required:"yes"`
+type commonMultiDeviceMixin struct {
+	Device        []string `long:"device" description:"encrypted device (can be more than one)" required:"yes"`
+	Authorization []string `long:"authorization" description:"authorization source (for each device, either 'keyring' or 'file:<key-file>')" required:"yes"`
 }
 
 type cmdAddRecoveryKey struct {
-	commonDeviceMixin
-	KeyFile string `long:"key-file" description:"path to recovery key file" required:"yes"`
+	commonMultiDeviceMixin
+	KeyFile string `long:"key-file" description:"path for generated recovery key file" required:"yes"`
 }
 
 type cmdRemoveRecoveryKey struct {
-	commonDeviceMixin
-	KeyFile string `long:"key-file" description:"path to recovery key file" required:"yes"`
+	commonMultiDeviceMixin
+	KeyFile []string `long:"key-file" description:"path to recovery key file(s) to be removed" required:"yes"`
```

Comment:
Should this now be `KeyFiles` (with an *s*) ?

Reply:
this question applies also for the fields in commonMultiDeviceMixin

Reply:
I think we do tend to use plural field names for multi-valued command arguments

Reply:
so devices/authorizations?